vocabularies/Org.OData.Temporal.V1.xml

```diff
@@ -67,98 +67,79 @@
+          <Annotation Term="Core.Description" String="If `true`, the period end is included in the time period" />
         </Property>
       </ComplexType>
 
-      <EnumType Name="QueryType" IsFlags="true">
-        <Member Name="TimeTravel" Value="1" />
-        <Member Name="TimeSeries" Value="2" />
-      </EnumType>
-
-
-      <!-- Metadata annotations that can also appear as instance annotations -->
-
-      <Term Name="From" Type="Edm.PrimitiveType" Nullable="false" AppliesTo="EntityType">
-        <Annotation Term="Core.Description" String="Lower boundary (inclusive) of application time. Date or DateTimeOffset value, or String with values min or now" />
-      </Term>
-
-      <Term Name="To" Type="Edm.PrimitiveType" Nullable="false" AppliesTo="EntityType">
-        <Annotation Term="Core.Description" String="Upper boundary (exclusive) of application time. Date or DateTimeOffset value, or String with values now or max" />
-      </Term>
-
-      <Term Name="SystemFrom" Type="Edm.PrimitiveType" Nullable="false" AppliesTo="EntityType">
-        <Annotation Term="Core.Description" String="Read-only lower boundary (inclusive) of system time. DateTimeOffset value or String with value min" />
-      </Term>
-
-      <Term Name="SystemTo" Type="Edm.PrimitiveType" Nullable="false" AppliesTo="EntityType">
-        <Annotation Term="Core.Description" String="Read-only upper boundary (exclusive) of system time. DateTimeOffset value or String with value max" />
-      </Term>
-
-
-      <!-- Instance annotations -->
-
-      <Term Name="History" Type="Collection(Edm.EntityType)" Nullable="false">
-        <Annotation Term="Core.Description" String="A collection of time slices representing the history of the annotated entity" />
-        <Annotation Term="Core.LongDescription" String="The time slices have the same entity type as the annotated entity and use the same select-expand projection." />
-      </Term>
+      <!-- TODO: move this to Core, where we already have QualifiedTypeName and QualifiedTermName? -->
+      <TypeDefinition Name="QualifiedActionName" UnderlyingType="Edm.String">
+        <Annotation Term="Core.Description" String="The qualified name of an action in scope." />
+      </TypeDefinition>
 
 
+      <!-- TODO: move to separate TemporalActions vocabulary? -->
       <!-- Operations on Temporal Objects -->
 
-      <Action Name="UpdateObject" IsBound="true" EntitySetPath="timeslices">
+      <Action Name="Update" IsBound="true" EntitySetPath="timeslices">
         <Annotation Term="Core.Description" String="Updates all time slices of a temporal object in a given period of application time after splitting partially overlapping time slices" />
         <Parameter Name="timeslices" Type="Collection(Edm.EntityType)" Nullable="false">
           <Annotation Term="Core.Description" String="Time slices to modify" />
         </Parameter>

```

Comment:
If `Employee_history` had a named entity set, then the action call `Employee_history_Set/Temporal.Update` would have the binding parameter `timeslices` contain more than one temporal object, but only one of them is updated. This should be made clearer in the text.

Reply:
Added to specification draft